release-notes/v1.1.0.md

```diff
@@ -0,0 +1,26 @@
+# HPE CSI Driver for Kubernetes 1.1.0 Release Notes
+
+## HPE CSI Driver for Kubernetes 1.1.0
+
+| **Version:** |1.1.0|
+|--------------|-----|
+| **Revision:** | Thursday March 12, 2020 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v1.1.0.
+
+|ID|Component |Title|
+|--|---------|-----|
+|CON-597|csi.k8s| Bind mounts are left around after pod removal.|
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v1.1.0.
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|CON-689|csi.k8s| After resize operation, new size is not reflected on PVC until its attached to a pod.|For devices with mount access type, our driver indicates to external resizer that filesystem resize is required. This causes FileSystemResizePending condition to be added to PVC. This condition is only cleared when PVC is attached to a pod and device/filesystem expansion is done by our node plugin. Only after this, new PVC size will be reflected. More information can be found at https://kubernetes-csi.github.io/docs/volume-expansion.html|None|
+|CON-690|csi.k8s| Clone of PVC with overridden parameters fails with status 400(BAD_REQUEST)| Certain properties like performancePolicy, dedupe cannot be changed on clone volume to differ from parent volume. These have to match with parent volume while creating a clone, both in storage class as well as PVC overrides.| Check storage class parameters and PVC overrides of clone to match with parent. These are applicable to properties like performancePolicy(with same block size) and dedupe settings, while others like IOPS can vary.|
```

Comment:
nit: like performancePolicy **and** dedupe cannot be changed on **the** cloned volume to differ from **the** parent volume.  These have to match with **the** parent volume while creating a clone, both in the storage class as well as **in the** PVC overrides.  Check **that the** storage class parameters and **the** PVC overrides of **the** clone **match those of the** parent **volume**.

Then add a space between performancePolicy and (